src/runtime_src/core/pcie/linux/shim.cpp

```diff
@@ -2199,6 +2199,15 @@ register_xclbin(const xrt::xclbin&)
   throw xrt_core::ishim::not_supported_error{__func__};
 }
 
+//Exec Buf with ctx handle.
+void
+shim::
+exec_buf(xclBufferHandle boh, const xrt::hw_context& hwctx)
```

Comment:
Let's not pass xrt::hw_context to shim.  I know I did this in one place, but it was a mistake.  Please pass xcl_hwctx_handle.

Reply:
fixed